ecf/hurricane/plots/jevs_hurricane_regional_tropcyc_plots.ecf

```diff
@@ -43,14 +45,18 @@ export STEP=plots
 export envir=dev
 export cyc=00
 export job=jevs_hurricane_regional_tropcyc_plots_${cyc}
+export PDY=20221231 
 
+#Set PDY to override setpdy.sh called in the j-jobs
+
+export PDY=20221231
 #Define the directory for TC-stats file 
-export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/1.0/hurricane_regional/tropcyc/stats
+export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/${evs_ver}/hurricane_regional/tropcyc/stats
```

Comment:
Change COMINstats to reflect the correct usage of EVS variables. Change to:
`export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/${evs_ver}/hurricane/regional/tropcyc/stats`
Note the "/" instead of "_" between hurricane and regional

Or better yet, use the variables:
`export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/${evs_ver}/${COMPONENT}/${RUN}/${VERIF_CASE}/${STEP}`